job/common/src/main/java/alluxio/job/plan/load/LoadConfig.java

```diff
@@ -32,15 +33,19 @@
   private static final long serialVersionUID = -7937106659935180792L;
   private final String mFilePath;
   private final int mReplication;
+  private final Set<String> mWorkerSet;
 
   /**
    * @param filePath the file path
    * @param replication the number of workers to store each block on, defaults to 1
+   * @param workerSet the worker set
    */
   public LoadConfig(@JsonProperty("filePath") String filePath,
-      @JsonProperty("replication") Integer replication) {
+      @JsonProperty("replication") Integer replication,
+      @JsonProperty("workerSet") Set<String> workerSet) {
     mFilePath = Preconditions.checkNotNull(filePath, "The file path cannot be null");
     mReplication = replication == null ? 1 : replication;
+    mWorkerSet = workerSet;
```

Comment:
should this make a copy of the set, instead of referring to the input set?

Reply:
Let's avoid the possibility to have `mWorkerSet` `null`?
```suggestion
    mWorkerSet = workerSet == null ? Collections.EMPTY_SET : new HashSet(workerSet) ;
```